client/src/json/json_template.html

```diff
@@ -70,4 +71,4 @@
 	<script src="jquery/jquery.slim.min.js"></script>
 	<script src="bootstrap/js/bootstrap.min.js"></script>
 </body>
-</html>
\ No newline at end of file
+</html>
```

Comment:
is this whitespace necessary?

Reply:
it's a newline and should have been there beforehand